dev/lint-python

```diff
@@ -23,6 +23,7 @@ PATHS_TO_CHECK="./python/pyspark/ ./examples/src/main/python/ ./dev/sparktestsup
 # TODO: fix pep8 errors with the rest of the Python scripts under dev
```

Comment:
I not sure if this is the last one, but I'm on mobile (yay Hawaii), but if it is we should change PATHS_TO_CHECK to wildcard include under ./dev/*.py